### PR TITLE
Add documentation and code standards

### PR DESCRIPTION
**Pull request by binaryoverload**

### CODE REVIEW

**Comment by binaryoverload** [comment]
@mrjvs I know you're not a java dev but if you could look over the templates/guidelines it'd be appreciated :D

**Comment by mrjvs** [comment]
Looks solid. No complaints.

File: src/main/java/com/cascadebot/cascadebot/messaging/MessageType.java [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
+import java.awt.Color;
 
 public enum MessageType {
 
-    INFO(Color.CYAN, "\u2139"), // ℹ
-    SUCCESS(Color.GREEN, "\u2705"), // ✅
-    WARNING(Color.YELLOW, "\u26A0"), // ⚠
-    MODERATION(Color.WHITE, "\uD83D\uDC6E"), // 👮
-    DANGER(Color.RED, "\u274C"), // ❌
+    INFO(Color.CYAN, "\u2139"), // Information icon
```
Comment (weeryan17) [inline src/main/java/com/cascadebot/cascadebot/messaging/MessageType.java]: I feel like we should also include the unicode character

File: src/main/java/com/cascadebot/cascadebot/utils/FormatUtils.java [hunk 2]
```diff
@@ -128,28 +128,28 @@ private static String getFormattedInlineFields(List<MessageEmbed.Field> fieldLis
...
 
         sb.append('`');
 
-        for(String head : header) {
+        for (String head : header) {
             sb.append(String.format("%-25s", head));
         }
 
-        sb.append("\u200B`");
+        sb.append("\u200B`"); // Zero width space
```
Comment (weeryan17) [inline src/main/java/com/cascadebot/cascadebot/utils/FormatUtils.java]: In reference to my previous statement having this just be text is fine as zero width space has no content

Comment (binaryoverload) [reply src/main/java/com/cascadebot/cascadebot/utils/FormatUtils.java]: Yeah but immediately looking at this you don't necessarily know it's a zero width space hence the comment

File: src/main/java/com/cascadebot/cascadebot/messaging/MessageType.java (hunk 1 above)
Comment (binaryoverload) [reply src/main/java/com/cascadebot/cascadebot/messaging/MessageType.java]: Okay

File: src/main/java/com/cascadebot/cascadebot/utils/FormatUtils.java (hunk 2 above)
Comment (weeryan17) [reply src/main/java/com/cascadebot/cascadebot/utils/FormatUtils.java]: Yegh having the text is fine. i was more referring to how we can't really put the unicode character here as zero with space is well 0 width

Comment (weeryan17) [reply src/main/java/com/cascadebot/cascadebot/utils/FormatUtils.java]: So I was saying ignore my previous review comment in this case

Comment (binaryoverload) [reply src/main/java/com/cascadebot/cascadebot/utils/FormatUtils.java]: Ah sorry for the misunderstanding, yeah I agree

**Comment by binaryoverload** [comment]
@weeryan17 have you looked over the guideline stuff (Especially the code style document)?

**Comment by weeryan17** [comment]
@binaryoverload yep

**Pull request merged**